### PR TITLE
Add "later" message to setup

### LINKED ISSUE #8: Change Later Message
Create a new widget that shows a message letting the user know they can change the settings they are choosing now later. Should be set at the bottom using a scaffold property.

### PR DESCRIPTION
**Pull request by Cal-Hagner**
This PR resolves #8.

This is how it looks:

image

### CODE REVIEW

File: lib/routes/setup/widgets/text.dart [hunk 1]
```diff
@@ -28,7 +29,31 @@ class SetupSubtitle extends StatelessWidget {
...
+      ),
+    );
+  }
+}
+
+class SetupFooter extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 40,
```
Comment (Matt-Gleich) [inline lib/routes/setup/widgets/text.dart:42]: Define height in the child container

File: lib/routes/setup/widgets/text.dart [hunk 2]
```diff
@@ -28,7 +29,31 @@ class SetupSubtitle extends StatelessWidget {
...
+      child: Center(
+        child: Container(
+          color: Colors.green,
```
Comment (gleich) [inline lib/routes/setup/widgets/text.dart]: Remove the color, make transparent

File: lib/routes/setup/widgets/text.dart (hunk 1 above)
Comment (gleich) [inline lib/routes/setup/widgets/text.dart]: Define height in the child container

File: lib/routes/setup/widgets/text.dart [hunk 3]
```diff
@@ -28,7 +29,31 @@ class SetupSubtitle extends StatelessWidget {
...
+          child: Center(
+            child: Text(
+              'Settings can be changed at any time',
```
Comment (gleich) [inline lib/routes/setup/widgets/text.dart:55]: Add setting icon in front using `Row` and `Icon`

**Review by gleich — CHANGES_REQUESTED** [review]
Please make the requested changes. I will be more clear in future issues with a picture of what it should look like in Figma.

**Comment by Cal-Hagner** [comment]
My apologies.

Here is the updated appearance:

Screenshot_1587074091

**Review by gleich — APPROVED** [review]

**Comment by gleich** [comment]
@Cal-Hagner No need to apologize. You didn't know!

**Pull request merged**